google-compute-lib.js

```diff
@@ -0,0 +1,245 @@
+const gcpCompute = require("@google-cloud/compute");
+const _ = require("lodash");
+require("@google-cloud/compute");
+const { ProjectsClient } = require("@google-cloud/resource-manager");
+const GCCompute = require("@google-cloud/compute");
+const parsers = require("./parsers");
+
+function getCredentials(params, settings) {
```

Comment:
Please reverse the functions declaration order with the outermost at the top and the innermost at the bottom, to improve code analysis.
I know eslint will complain when a function is used before its declaration, but we should add a rule to ignore that.